gem/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+from gem.constants import GENDERS
+
+
+class RegistrationForm(forms.Form):
+    username = forms.RegexField(
+        regex=r'^[\w.@+-]+$',
+        widget=forms.TextInput(
+            attrs=dict(
+                required=True,
+                max_length=30,
+            )
+        ),
+        label=_("Username"),
+        error_messages={
+            'invalid': _("This value must contain only letters, "
+                         "numbers and underscores."),
```

Comment:
I think this regex was build for email addresses? Either way, if the user follows the error message they'll get a valid username, so I'm okay with leaving it for now.